conf/Caddyfile.in

```diff
@@ -8,6 +8,20 @@
 	gzip {
 		not /stream
 	}
+
+header /stream {
```

Comment:
Use `header /stream/stream.` for this one so it only picks up the stream.[num].ts files with the encoded audio data for HLS. Apparently Caddy treats these as straight prefixes to the path.